ledger/ledger-api-domain/BUILD.bazel

```diff
@@ -23,6 +24,7 @@ da_scala_library(
   srcs = glob(['src/test/**/*.scala'], exclude = ['src/test/**/*Spec.scala', 'src/test/**/*Test.scala']),
   deps = [
     ':ledger-api-domain',
+    '//daml-lf/data',
```

Comment:
@francesco-da @gaboraranyossy-da 
Can I do that ?

Reply:
yep, that's fine